http1_server.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+import socket
+import email.utils
+
+
+def response_ok(request):
+    response_code = "HTTP/1.1 200 OK"
+    date = "Date: {}".format(email.utils.formatdate(usegmt=True))
+    headers = "Content-Type: Text/HTML"
+    response = "{}\r\n{}\r\n{}\r\n\r\n".format(response_code, date, headers)
```

Comment:
this is rigid, and thus brittle.  What happens when you want to add a new header line beyond the Content-Type header that is there now? 

It's worth noting that what you have bound to the symbol `headers` is only one HTTP header, and the value you've bound to the symbol `date` is another HTTP header, so your naming indicates you aren't quite clear on the concept yet (or perhaps the naming is keeping the concept from clarifying for you?)

On the other hand, the structure of the response is absolutely correct, so good job there.